### PR TITLE
feat(theme): Adds readonly input theme

### PR DESCRIPTION
**Pull request by talend-glorieux**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
The readonly input fields are rendered as regular input fields with the
editing disabled.

**What is the new behavior?**
They now behave more like disabled input fields without the focused colors.

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://408.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://408.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://408.talend.surge.sh)

File: packages/theme/src/theme/_forms.scss [hunk 1]
```diff
@@ -55,7 +55,7 @@ form {
 		border-radius: 0;
 		font-size: $font-size-base;
 
-		&:focus {
+		&:not(:read-only):focus {
```
Comment (frassinier) [inline packages/theme/src/theme/_forms.scss]: read-only input could not be focused?

File: packages/theme/src/theme/_forms.scss [hunk 2]
```diff
@@ -31,7 +31,7 @@ form {
 		[type=email].form-control,
 		[type=tel].form-control,
 		[contenteditable].form-control {
-			&:focus + label {
+			&:not(:read-only):focus + label {
```
Comment (frassinier) [inline packages/theme/src/theme/_forms.scss]: read-only input could not be focused?

File: packages/theme/example/index.html [hunk 3]
```diff
@@ -895,6 +895,16 @@ <h1 id="forms">Forms</h1>
                   <label class="control-label" for="disabledInput">Disabled input</label>
                 </div>
 
+                <div class="form-group">
+                  <input class="form-control" id="readonlyInput" type="text" placeholder="Readonly input here..." readonly="">
+                  <label class="control-label" for="readonlyInput">Readonly input</label>
+                </div>
+
+                <div class="form-group">
+                  <input class="form-control" id="readonlyInput" type="text" value="Simple Input" readonly="">
```
Comment (frassinier) [inline packages/theme/example/index.html]: Please use another `id`

**Pull request closed without merging**